src/party_menu.c

```diff
@@ -409,6 +412,9 @@ static bool8 SetUpFieldMove_Dive(void);
 #include "data/pokemon/tutor_learnsets.h"
 #include "data/party_menu.h"
 
+// Text string printed when changing the form of certain species like Shaymin and Giratina
+const u8 ChangedForm[] = _("{STR_VAR_1} changed Forme!{PAUSE_UNTIL_PRESS}");
```

Comment:
This text string was updated somewhere inbetween Gen. 4 and Gen. 7. In [the latter](https://youtu.be/jQBBXX6c6VA?t=173), the string when changing either Shaymin's or Giratina's form actually says "Pokémon transformed!".